Gems/AudioSystem/Code/Source/Engine/AudioInternalInterfaces.h

```diff
@@ -769,124 +769,6 @@ namespace Audio
             return (eStatus == eARS_SUCCESS || eStatus == eARS_FAILURE);
         }
 
-#if !defined(AUDIO_RELEASE)
-        // Debug Logging Helper
-        AZStd::string ToString()
-        {
-            static const AZStd::unordered_map<const EAudioManagerRequestType, const AZStd::string> managerRequests
-            {
```

Comment:
This code would have crashed on shutdown if it was ever called due to the static `AZStd::unordered_map`
It would have been using the AZ Allocators after they have been un-attached from the gem module

Reply:
Yeah it's bad, I don't care about removing it.  It helped me out at one point but that was a long time ago.